src/Swarrot/Processor/SignalHandler/SignalHandlerProcessor.php

```diff
@@ -67,7 +63,7 @@ public function process(Message $message, array $options)
     {
         $return = $this->processor->process($message, $options);
 
-        if (extension_loaded('pcntl') && $this->shouldStop($options)) {
```

Comment:
I don't understand this code. IMHO this is wrong. We should always return `$return`, isn't ?

Reply:
Right!